requirements-cuda.txt

```diff
@@ -7,3 +7,4 @@ nvidia-ml-py # for pynvml package
 torch == 2.3.0
 xformers == 0.0.26.post1  # Requires PyTorch 2.3.0
 vllm-flash-attn == 2.5.8.post2  # Requires PyTorch 2.3.0
+compressed-tensors == 0.3.3
```

Comment:
I don't think we can have this as a dependency until we remove the transformers dep from compressed tensors